safehttp/handler.go

```diff
@@ -14,12 +14,25 @@
 
 package safehttp
 
-// Handler TODO
+// Handler responds to an HTTP request.
+//
+// ServeHTTP should set headers, call one of the writing functions on the ResponseWriter
+// to write the body of the response and then return the Result received from writing
+// to the ResponseWriter.
+//
+// It is not valid to use the ResponseWriter or read from the body of the IncomingRequest
+// after or concurrently with the completion of the ServeHTTP call.
```

Comment:
I don't understand this.

Reply:
We copied this pretty much straight off from the godoc of net/http.Handler. I agree it's pretty confusing so I'll remove it.